.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
is this critical for windows CI support? If not mind removing it and submitting a new PR?